common/models/event.json

```diff
@@ -19,7 +19,7 @@
       "type": "string"
     },
     "location": {
-      "type": "string",
```

Comment:
@darrenrichmond there was one small typo in the json in your last changes. I have gone ahead and fixed it, and things seem to be working correctly now, so I'll go ahead and merge it.